apps/opencs/model/world/columns.cpp

```diff
@@ -256,6 +287,11 @@ int CSMWorld::Columns::getId (const std::string& name)
 
 namespace
 {
+    static const char *sSkills[] =
```

Comment:
Looks like something is missing here. But also possible duplication. We must already have the list of skills elsewhere (as a separate record table IIRC).